### PR TITLE
Docker image update to 1.7.0-cuda11.0-cudnn8

### PR DESCRIPTION
**Pull request by vfdev-5**
* Fixed Apex installation

@fco-dv please update pytorch version

Check list:
* [ ] New tests are added (if a new feature is added)
* [ ] New doc strings: description and/or example code are in RST format
* [ ] Documentation is updated (if required)

### CODE REVIEW

**Comment by vfdev-5** [comment]
@fco-dv let's remove all those default values as it leads to a lot of confusion: from bash and in dockerfiles.

To set up pytorch version it is here (single entry):
- https://github.com/pytorch/ignite/blob/3affd95f9480b4f6a9070d800d8505bc3e80e0dc/.circleci/config.yml#L23 
Let's also print an error in bash if env var is not defined.

File: docker/hvd/build_all.sh [hunk 1]
```diff
@@ -30,7 +30,14 @@ cd $curr_dir/hvd
 set -eu
 
 image_tag=""
-pth_version=${PTH_VERSION:-1.6.0-cuda10.1-cudnn7}
+
+if [[ -z "${PTH_VERSION}" ]]; then
+    echo "PTH_VERSION is not set"
+    exit 1
+else
```
Comment (vfdev-5) [inline docker/hvd/build_all.sh:37]: We can define `pth_version=${PTH_VERSION}` without else for better understanding.

File: docker/hvd/build_all.sh [hunk 2]
```diff
@@ -30,7 +30,14 @@ cd $curr_dir/hvd
...
+    pth_version=${PTH_VERSION}
+fi
+
 hvd_version=${HVD_VERSION:-v0.21.0}
```
Comment (vfdev-5) [inline docker/hvd/build_all.sh:41]: Let's remove default values for HVD_VERSION and MSDP_VERSION too

File: docker/hvd/build_all.sh (hunk 1 above)
Comment (fco-dv) [reply docker/hvd/build_all.sh:37]: yes of course

File: docker/hvd/build_all.sh (hunk 2 above)
Comment (fco-dv) [reply docker/hvd/build_all.sh:41]: yes indeed

**Comment by vfdev-5** [comment]
@fco-dv I asked your review as this PR is initialized by myself and I can not approve my own PR :)

**Review by fco-dv — APPROVED** [review]
LGTM

**Pull request merged**